### PR TITLE
Set favicon for IE

### PR DESCRIPTION
**Pull request by SviridovSV**
Need set favicon for IE browsers and Android
photo_2018-05-24_13-13-14

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   "density": "3.0"
+  },
+  {
+   "src": "\/android-icon-192x192.png",
+   "sizes": "192x192",
+   "type": "image\/png",
+   "density": "4.0"
+  }
+ ]
+}
```
Comment (andriyParashchuk) [inline manifest.json]: What does this file do?

**Comment by andriyParashchuk** [comment]
No need to create so many favicons
I think need add attribute for IE browsers, and check this on real IE browsers

File: manifest.json (hunk 1 above)
Comment (SviridovSV) [reply manifest.json]: This file will be removed in next commit.

**Comment by SviridovSV** [comment]
@andriyParashchuk  Do you plan to check staging on real IE?

**Comment by SviridovSV** [comment]
staging: http://jt-pr333.surge.sh

File: _includes/head.html [hunk 2]
```diff
@@ -41,11 +41,14 @@
     <meta name="twitter:site" content="@jetthoughts">
     <meta name="twitter:image" content="{{ site.url }}/social-logo.png">
 
+
```
Comment (andriyParashchuk) [inline _includes/head.html]: delete line

**Comment by andriyParashchuk** [comment]
@SviridovSV 
- why you deleted apple-touch-icon-144-precomposed.png? We need this icon.
- Why you deleted favicon_32.png and favicon_16.png? You can just rename they. Old icons have less size

**Comment by andriyParashchuk** [comment]
On IE browsers looks great



android browsers also
screenshot_2018-06-01-20-15-16-761_com android chrome

You need:
- return apple-touch-icon-144-precomposed
- return old icons and rename they
- squash commits and change to https://github.com/jetthoughts/how-we-work#gitgithub

File: _includes/head.html (hunk 2 above)
Comment (SviridovSV) [reply _includes/head.html]: Done

**Review by andriyParashchuk — COMMENTED** [review]
LGTM

**Pull request merged**